homework/homework.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
```

Comment:
`'use strict'` is implied automatically by the ES Module Loader. It's needed only for CommonJS modules.